util/internal/integrationtest/v5migrationhelper/src/main/java/org/hibernate/search/test/util/impl/ExpectedLog4jLog.java

```diff
@@ -10,25 +10,24 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
-import org.apache.log4j.AppenderSkeleton;
-import org.apache.log4j.Level;
-import org.apache.log4j.Logger;
-import org.apache.log4j.spi.LoggingEvent;
+import org.junit.rules.TestRule;
+import org.junit.runners.model.Statement;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.Logger;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.layout.PatternLayout;
 import org.hamcrest.CoreMatchers;
 import org.hamcrest.Description;
 import org.hamcrest.Matcher;
 import org.hamcrest.StringDescription;
 import org.hamcrest.TypeSafeMatcher;
-import org.junit.rules.TestRule;
-import org.junit.runners.model.Statement;
 
-/**
- * @author Yoann Rodiere
- */
 public class ExpectedLog4jLog implements TestRule {
```

Comment:
I think you replaced the code with the other implementation, `org.hibernate.search.util.impl.test.rule.ExpectedLog4jLog`? If so, I'd say we may as well remove this class and replace it with the other implementation everywhere. I can do it if you want?